src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/AggregateProjectCapabilityResolver.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+
+namespace Microsoft.VisualStudio.Razor;
+
+[Export(typeof(AggregateProjectCapabilityResolver))]
+internal sealed class AggregateProjectCapabilityResolver
+{
+    private readonly IEnumerable<IProjectCapabilityResolver> _projectCapabilityResolvers;
+
+    [ImportingConstructor]
+    public AggregateProjectCapabilityResolver([ImportMany] IEnumerable<IProjectCapabilityResolver> projectCapabilityResolvers)
+    {
+        _projectCapabilityResolvers = projectCapabilityResolvers;
+    }
+
+    public bool HasCapability(object project, string capability)
```

Comment:
Looks like this is never called, so can be removed, and the corresponding method removed from the interface?

Reply:
Removed